src/include/mpir_refcount_global.h

```diff
@@ -25,6 +25,16 @@ typedef volatile int MPIR_cc_t;
         MPIR_Assert(*(incomplete_) >= 0);       \
     } while (0)
 
+#define MPIR_cc_inc(cc_ptr_) \
+    do { \
+        (*(cc_ptr_))++; \
+    } while (0)
+
+#define MPIR_cc_dec(cc_ptr_) \
+    do { \
+        (*(cc_ptr_))--; \
+        MPIR_Assert(*(cc_ptr_) >= 0); \
+    } while (0)
```

Comment:
These are definitely welcome. The naming might be a bit too subtle. Maybe once these are in we can propose renaming the old ones to something more descriptive like `fetch_and_inc` or something.

Reply:
Since the compiler will catch it if we mix one for another and there isn't much of a semantic difference here, so I tend to not worry about it.